tensorboard/webapp/metrics/views/right_pane/settings_view_component.ng.html

```diff
@@ -40,7 +40,10 @@ <h3 class="section-title">General</h3>
     </mat-checkbox>
     <span class="indent">(Scalars only)</span>
     <div class="indent range-selection" *ngIf="isRangeSelectionAllowed">
-      <mat-checkbox [disabled]="!isAxisTypeStep()"
+      <mat-checkbox
+        [checked]="isScalarStepSelectorRangeEnabled"
+        (change)="stepSelectorRangeToggled.emit()"
+        [disabled]="!isAxisTypeStep()"
```

Comment:
Is it worth adding a test that the checkbox is disabled when there is a different axis type?

Reply:
Yea good idea. Added.